dataeng/resources/warehouse-transforms-master-ci.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup to run python script to create snowflake schema
+cd $WORKSPACE/analytics-tools/snowflake
+make requirements
+
+# Setup to run dbt commands
+cd $WORKSPACE/warehouse-transforms
+
+COMMIT_ID=$(git rev-parse --short HEAD)
+# Only run CI on merge commits, otherwise exit
+HEAD_COMMIT=$(git rev-parse HEAD)
+LAST_MERGE_COMMIT=$(git log --merges origin/master --format='%H' --max-count=1)
+if [ $HEAD_COMMIT == $LAST_MERGE_COMMIT ]
+then
+    echo "This is one of merge commit, Run CI"
+else
+    echo "Exiting because not a merge commit"
+    exit 0
+fi 
+
+
+# Get second last merge commit id and compares it with the HEAD to find (git diff) files changed.
+PREV_MERGE_COMMIT_ID=$(git log --merges origin/master --format='%H' --max-count=2 | sed -n 2p)
+
+git diff $PREV_MERGE_COMMIT_ID --name-only
```

Comment:
Comparing difference between the HEAD and second last merge commit to find out the projects files have changed.